js/kucoinfutures.js

```diff
@@ -443,11 +436,28 @@ module.exports = class kucoinfutures extends kucoin {
                 symbol = symbol + '-' + this.yymmdd (expiry, '');
                 type = 'future';
             }
-            const baseMinSizeString = this.safeString (market, 'baseMinSize');
-            const quoteMaxSizeString = this.safeString (market, 'quoteMaxSize');
             const inverse = this.safeValue (market, 'isInverse');
             const status = this.safeString (market, 'status');
             const multiplier = this.safeString (market, 'multiplier');
+            const contractSize = this.parseNumber (Precise.stringAbs (multiplier));
+            const tickSize = this.safeNumber (market, 'tickSize');
+            const lotSize = this.safeNumber (market, 'lotSize');
+            const maxQuantity = this.safeNumber (market, 'maxOrderQty');
+            let limitAmountMin = lotSize;
+            if (limitAmountMin === undefined) {
+                limitAmountMin = this.safeNumber (market, 'baseMinSize');
+            }
+            let limitAmountMax = maxQuantity;
+            if (limitAmountMax === undefined) {
+                limitAmountMax = this.safeNumber (market, 'baseMaxSize');
+            }
+            let limitPriceMin = tickSize;
```

Comment:
Do you use VSCode, and do you have [eslint](dbaeumer.vscode-eslint) installed? I got a warning here that this should be `const` intead of `let`

Reply:
don't worry about changing it, i've got it